precompiles/gov/integration_test.go

```diff
@@ -475,5 +477,203 @@ var _ = Describe("Calling governance precompile from EOA", func() {
 				Expect(out.TallyResult.NoWithVeto).To(Equal("0"))
 			})
 		})
+
+		Context("proposal query", func() {
+			method := gov.GetProposalMethod
+			BeforeEach(func() {
+				callArgs.MethodName = method
+			})
+
+			It("should return a proposal", func() {
+				callArgs.Args = []interface{}{uint64(1)}
+				txArgs.GasLimit = 200_000
```

Comment:
I believe the gas limit is set to avoid the estimate right? In this case I would set it once [here](https://github.com/evmos/evmos/blob/19e710e55bb33c28ec56dfea9a4a4dd80c5cdfbd/precompiles/gov/integration_test.go#L74-L74) so we don't have to do it in all tests